src/apis/icronjob_management_api.ts

```diff
@@ -1,6 +1,8 @@
 import {IIdentity} from '@essential-projects/iam_contracts';
 
+import {Subscription} from '@essential-projects/event_aggregator_contracts';
 import {CronjobConfiguration, CronjobHistoryEntry} from '../data_models/cronjob/index';
+import {Messages} from '..';
```

Comment:
Wenn aus Ordnern importiert wird, bitte immer ein `index` hinten dran hängen.

Wir hatten des öfteren schon Probleme mit dem Bundling im Studio, weil das gefehlt hat.
(Außerdem sieht man so sofort, dass hier aus einem Ordner importiert wird und nicht direkt aus einer Datei).

```suggestion
import {Messages} from '../messages/index';
```